infra/app/service/main.tf

```diff
@@ -88,7 +86,7 @@ module "service" {
 
   db_vars = module.app_config.has_database ? {
     security_group_ids = data.aws_rds_cluster.db_cluster[0].vpc_security_group_ids
-    access_policy_arn  = data.aws_iam_policy.db_access_policy[0].arn
```

Comment:
we need to pass in the app db access policy and migrator db access policy to the service module